examples/nlp/semantic_similarity_with_keras_nlp.py

```diff
@@ -0,0 +1,263 @@
+"""
```

Comment:
I think you pulled in your other guide by accident, you may want to rebase out the commits for this guide.